### PR TITLE
MKC 8 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/ransom_note.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Ransom Note
+ManaCost:1
+Types:Artifact Clue
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSurveil | TriggerDescription$ When CARDNAME enters the battlefield, surveil 1.
+SVar:TrigSurveil:DB$ Surveil | Amount$ 1
+A:AB$ Charm | Cost$ 2 Sac<1/CARDNAME> | Choices$ DBCloak,DBGoad,DBDraw
+SVar:DBCloak:DB$ Cloak | Defined$ TopOfLibrary | RememberCloaked$ True | SpellDescription$ Cloak the top card of your library.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ransom_note.txt:7]: why remember?

File: forge-gui/res/cardsfolder/upcoming/tesak_judiths_hellhound.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Tesak, Judith's Hellhound
+ManaCost:3 R
+Types:Legendary Creature Elemental Dog
+PT:3/3
+K:Unleash
+S:Mode$ Continuous | Affected$ Dog.Other+YouCtrl | AddKeyword$ Unleash | Description$ Other Dogs you control have unleash.
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+HasCounters | AddKeyword$ Haste | Description$ Creatures you control with counters on them have haste.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigMana | TriggerDescription$ Whenever CARDNAME attacks, add {R} for each attacking creature.
+SVar:TrigMana:DB$ Mana | Produced$ R | Produced$ R | Amount$ X
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tesak_judiths_hellhound.txt:9]: fix params

File: forge-gui/res/cardsfolder/upcoming/trouble_in_pairs.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Trouble in Pairs
+ManaCost:2 W W
+Types:Enchantment
+R:Event$ BeginTurn | ActiveZones$ Battlefield | ValidPlayer$ Opponent | ExtraTurn$ True | Skip$ True | Description$ If an opponent would begin an extra turn, that player skips that turn instead.
+T:Mode$ AttackersDeclared | AttackingPlayer$ Opponent | CheckSVar$ X | SVarCompare$ GE2 | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever an opponent attacks you with two or more creatures, draws their second card each turn, or casts their second spell each turn, you draw a card.
+SVar:X:Count$ValidAll Creature.attackingYou
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/trouble_in_pairs.txt:6]: where are you getting such weird ValidAll ideas? :P
use IsPresent and don't forget NoResolvingCheck please...

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/trouble_in_pairs.txt]: Actually you can just use ValidAttackers + ValidAttackersAmount for a cleaner result

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/trouble_in_pairs.txt]: Unless I scripted it wrong it gave an out of bounds error when I was attack by one creature. `Range [2, 1) out of bounds for length 1`

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/trouble_in_pairs.txt]: ugh I'll check

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/trouble_in_pairs.txt]: well `ValidAttackers$ Creature.attackingYou | ValidAttackersAmount$ GE2` works fine for me

**Review by tool4ever — APPROVED** [review]

**Pull request merged**